.clang-tidy

```diff
@@ -37,7 +37,7 @@ Checks: '-*,
   -readability-inconsistent-declaration-parameter-name,
   readability-redundant-control-flow,
   readability-redundant-smartptr-get,
-  -readability-string-compare'
+  readability-string-compare'
```

Comment:
github is being weird. Locally, when I do a `git diff master`, this does not show up and my master is in sync with upstream/master. I guess this will go away once I squash all commits.